pkg/webhook/patch.go

```diff
@@ -182,6 +183,42 @@ func addEnvironmentVariable(pod *corev1.Pod, envName, envValue string) patchOper
 	return patchOperation{Op: "add", Path: path, Value: value}
 }
 
+func addCommandArgs(pod *corev1.Pod, app *v1beta1.SparkApplication) []patchOperation {
+	var podSpec *v1beta1.SparkPodSpec
```

Comment:
This can be moved to after line 198.

Reply:
@liyinan926 

Fixed, please double check.